README.md

```
@@ -85,59 +85,62 @@ Astrocelerate is developed with the belief that humanity's greatest frontier sho
 Astrocelerate made its first commit on November 28, 2024. As of December 28, 2025, it has been in development for 395 days. During its first two months in existence, Astrocelerate was written in OpenGL, but it henceforth migrated to Vulkan due to inherent limitations with OpenGL's state-machine programming paradigm.
 
 ## Screenshots
-The following screenshots document the development of Astrocelerate (Date format: DD/MM/YYYY).
+The following screenshots document the development of Astrocelerate (Date format: YYYY-MM-DD).
 
-### 17/12/2025
+### 2026-04-12
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)